jekyll/_cci2/github-integration.adoc

```diff
@@ -335,6 +335,12 @@ Once you have set up your first pipeline, you will be able to add triggers to it
 
 NOTE: Only organization admins and users that have admin access to at least one repository in an organization can install the GitHub App.
 
+Once the GitHub App is installed, it is recommend that each user authorize the GitHub App connection. To do this, users can navigate to **Project Settings > Pipelines > Add Pipeline > connections** and clicking the **"Authorize"** button.
```

Comment:
Once the GitHub App is installed, each user should authorize the GitHub App connection.
To do this, each user should navigate to menu:Project Settings[Pipelines>Add Pipeline>Connection] and select btn:[Authorize].